Index: packagekit/client/pk-offline-update.c
===================================================================
--- packagekit.orig/client/pk-offline-update.c
+++ packagekit/client/pk-offline-update.c
@@ -160,7 +160,7 @@ pk_offline_update_progress_cb (PkProgres
 			 * advise of the new percentage completion when installing updates */
 			msg = g_strdup_printf ("%s - %i%%", _("Installing Updates"), percentage);
 		}
-		if (percentage > 10)
+		if (percentage >= 0)
 			pk_offline_update_set_plymouth_msg (msg);
 
 		/* print on terminal */
